test/infrastructure/docker/config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-capi-docker/capd-manager:dev
+      - image: gcr.io/k8s-staging-capi-docker/capd-manager-amd64:dev
```

Comment:
I don't this should be part of the commit

Reply:
Ah, good catch, stung again by `git commit -a`